backend/src/main/nl/tudelft/ti2806/riverrush/game/TeamFullException.java

```diff
@@ -0,0 +1,7 @@
+package nl.tudelft.ti2806.riverrush.game;
+
+/**
+ * Is thrown if all the teams are full..
+ */
+public class TeamFullException extends Throwable {
```

Comment:
This should be an unchecked exception since it does not need handling. Please extend from RuntimeException.